ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.74__ocp-4-13.yaml

```diff
@@ -0,0 +1,71 @@
+base_images:
+  ubi-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: apollo-ci
+    namespace: stackrox
+    tag: stackrox-test-0.3.57
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.13.0-0
+        upper: 4.14.0-0
+resources:
+  '*':
+    requests:
+      cpu: 2000m
+      memory: 4000Mi
+test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
+tests:
+- as: qa-e2e-tests
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      TEST_SUITE: ocp-qa-e2e-tests
```

Comment:
for `3.74` you'll need an `openshift-4-qa-e2e-tests` `TEST_SUITE` to match the jobs:
```
[stackrox (release-3.74) (admin)]$ ls -l scripts/ci/jobs/openshift_4*
-rwxr-xr-x 1 gavin gavin 497 Apr 24 11:39 scripts/ci/jobs/openshift_4_operator_e2e_tests.py
-rwxr-xr-x 1 gavin gavin 512 Apr 24 11:39 scripts/ci/jobs/openshift_4_qa_e2e_tests.py
```

Reply:
Oh, because those changes didn't get into the release branches? That makes sense